plugins/hypervisors/vmware/src/com/cloud/hypervisor/vmware/manager/VmwareManagerImpl.java

```diff
@@ -707,6 +709,22 @@ private String setupMountPoint(String parent) {
         return mountPoint;
     }
 
+    @Override
+    public synchronized void setSystemVmTmpltPermission(String mountPoint) {
+        if (!s_isSystemVmTmpltPermissionSet) {
+            s_logger.debug("Set permissions for " + mountPoint);
+            String result = null;
+            Script script = new Script(true, "chmod", _timeout, s_logger);
+            script.add("-R", "0777", mountPoint);
```

Comment:
@sureshanaparti In place of using 0777, I think it is better to declare a constant and use it.